odl/discr/discr_mappings.py

```diff
@@ -444,13 +425,14 @@ def nearest(arg, out=None):
 
     def __repr__(self):
         """Return ``repr(self)``."""
-        inner_str = '\n  {!r},\n  {!r},\n  {!r}'.format(self.range,
-                                                        self.grid,
-                                                        self.domain)
-        if self.order == 'F':
-            inner_str += ",\n  order='F'"
-        if self._variant == 'right':
-            inner_str += ",\n  variant='right'"
+        inner_str = '\n  {!r},\n  {!r},\n  {!r}'.format(
+            self.range, self.grid, self.domain)
+        sep = ',\n '
```

Comment:
Agree, we have some hardcoded stuff there. Will have a look into the util. Could be handy to have something additional like `long_members` which break like before and after.